src/servers/src/http/header.rs

```diff
@@ -17,7 +17,7 @@ use headers::{Header, HeaderName, HeaderValue};
 pub const GREPTIME_DB_HEADER_FORMAT: &str = "x-greptime-format";
 pub const GREPTIME_DB_HEADER_EXECUTION_TIME: &str = "x-greptime-execution-time";
 
-pub static GREPTIME_DB_HEADER_NAME: HeaderName = HeaderName::from_static("x-greptime-name");
+pub static GREPTIME_DB_HEADER_NAME: HeaderName = HeaderName::from_static("x-greptime-db-name");
```

Comment:
@tisonkun We notice that you change the key name from `x-greptime-db-name` to `x-greptime-name` in https://github.com/GreptimeTeam/greptimedb/pull/3062. Is there a specific reason to do so? We have online systems that already rely on this header key name.

Reply:
@killme2008 we may have a decision here and for previous header that already leaks, add a special comment not to change.

Reply:
`dbname` is a concept we expose to our user and is being referred in our doc website. `x-greptime-name` on the other hand seems to be the name of the whole cluster, which is not really the case here. But it's always open for discussion as long as we update all the dependencies together 😆

Reply:
I prefer `x-greptime-db` or `x-greptime-dbname` personally. Let's revert it currently, and do another refactor finally.

Reply:
Yeah. The most important thing is that we should be aware that HeaderNames are user-facing **public interfaces**. We'd better manage them consistently together and using a consistent flavor so that users can infer the names.

Reply:
Exactly